### DISCUSSION TITLE
reverse_image_search is not working after indexing large amount pictures

### DISCUSSION THREAD

**Discussion by ghost** [milvus-io/milvus#16945] [Q&A and General discussion]
各位专家 我们碰到一个问题
在compose up好这个项目
https://github.com/milvus-io/bootcamp/tree/master/solutions/reverse_image_search/quick_deploy
并且添加了五万张图片之后 搜索不到任何图片怎么办
下面是日志文件
[all.log.gz](https://github.com/milvus-io/milvus/files/8676032/all.log.gz)

**Comment by hyyz17200** [comment]
@kimchichef   are working together with me to try this reverse_image_search Bootcamp. 

I was setting up a fresh Debian 11 VM. After I followed the readme, the Bootcamp started without issue. 

Then I loaded 16 images as a quick test on the built-in WebUI. So far, so good. The indexing and searching function well.

After that, I added more pictures. Like @kimchichef  said, it was a large folder containing over 50000 images. 

The indexing looks normal, and it runs for about 8 hours—no errors throughout the whole indexing process.

However, searching failed. And I couldn't figure out how this issue happened. 

Please check the log above. I do appreciate any help. Thanks!
